Assets/Plugins/iOS/Shopify/WebCheckoutSession.swift

```diff
@@ -74,3 +78,12 @@ private extension UnityAppController {
         return unityController.rootViewController
     }
 }
+
+// Small helper class for presenting the SFSafariViewController
+private class WebCheckoutNavigationController: UINavigationController {
+        
+    // Only allow the same rotations the underlying game allows.
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return UnityAppController.root.supportedInterfaceOrientations
```

Comment:
We can also use `GetAppController().rootViewController` from `UnityAppController.h` since we include that class in the bridging header